lib/internal/worker.js

```diff
@@ -438,7 +440,12 @@ class Worker extends EventEmitter {
   }
 }
 
-class InternalWorker extends Worker {}
+class InternalWorker extends Worker {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(filename, options) {
+    super(filename, options);
+  }
```

Comment:
why is this necessary?

Reply:
Because V8 has not implemented https://github.com/tc39/ecma262/pull/2216:

```js
delete Array.prototype[Symbol.iterator];
new class extends class {} {}
```